docs/lifecycle.md

```diff
@@ -70,35 +65,29 @@ The GHG Protocol provides two methods for tracking Scope 2 emissions: the market
 
 - The location-based method: calculates emissions based on the emissions intensity of the local grid area where the electricity usage occurs.
 
-| Topic | DIMPACT | SRI | GroupM / EY | Scope3
-|---| --- | --- | --- | ---
-| Requires location-based emissions | Recommended | No | Recommended | Yes
+| Topic                             | DIMPACT     | SRI | GroupM      |
+| --------------------------------- | ----------- | --- | ----------- |
+| Requires location-based emissions | Recommended | No  | Recommended |
 
 ### Ad Tech Supply Chain
 
 The ad tech supply chain is a critical component to understand emissions from media and advertising companies. It is opaque and has multiple layers of intermediaries, making it difficult to fully understand.
 
-| Topic | DIMPACT | SRI | GroupM / EY | Scope3
-|---| --- | --- | --- | ---
-| Includes emissions from programmatic vendors | No | No | Yes | Yes
-| Includes emissions from data and identity providers | No | No | No | Yes
-| Includes emissions from technology embedded into creatives | No | No | Yes | Yes
+DIMPACT is omitted as its methodology does not currently include ad selection.
+
+| Topic                                                      | SRI | GroupM |
+| ---------------------------------------------------------- | --- | ------ |
+| Includes emissions from programmatic vendors               | No  | Yes    |
+| Includes emissions from data and identity providers        | No  | No     |
+| Includes emissions from technology embedded into creatives | No  | Yes?   |
```

Comment:
not clear what `Yes?` means. a small comment indicating why this isn't clear based on their report would help.

Reply:
will change to "unspecified"